dev-tools/scripts/scriptutil.py

```diff
@@ -66,14 +70,19 @@ def on_or_after(self, other):
            (self.bugfix > other.bugfix or self.bugfix == other.bugfix and
            self.prerelease >= other.prerelease)))
 
+  def gt(self, other):
```

Comment:
This is used to check if a given version is greater than current. Will be used in releaseWizard script to be committed later